roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -227,7 +227,7 @@ pminio(){
         sudo bash -c "sed -i '/DEFAULT_FILE_STORAGE/d' /etc/pulp/settings.py"
         sudo bash -c "sed -i '/MEDIA_ROOT/d' /etc/pulp/settings.py"
         sudo bash -c "sed -i '/AWS_S3_ENDPOINT_URL/d' /etc/pulp/settings.py"
-        podman stop pulp_minio
+        podman stop pulp_minio && podman rm pulp_minio
```

Comment:
On second thought, will this rm the pulp_minio container if it's not running? I think we want to do that.

Reply:
Actually I tested this out and it works. `podman stop pulp_minio` returns 0 even if the container is stopped.

Reply:
FYI, you can do `podman rm --force pulp_minio` instead to do both actions.